backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/database/repos/RegionsRepository.kt

```diff
@@ -25,4 +25,12 @@ object RegionsRepository {
 
     fun findByIds(ids: List<Int>) =
         RegionEntity.find { Regions.id inList ids }.sortByKeys({ it.id.value }, ids)
+
+    fun findRegionById(regionId: Int): RegionEntity {
+        return RegionEntity[regionId]
+    }
+
+    fun updateDataPolicy(region: RegionEntity, dataPrivacyText: String) {
```

Comment:
Check length of dataPrivacyText here to avoid errors in the DB.

Reply:
Is it fine to restrict in the frontend textarea? Could add a char counter below

Reply:
I think we might also want to check this constraint (and conditionally throw) here, so that we catch this error instead of relying on the DB / Exposed to throw an error.

Reply:
Yes, we should have a check in the graphql schema, else attackers are able to send megabytes of text to the database instead of only 20k chars.

Reply:
You can just throw a RuntimeException if the text is too long.

Reply:
added a max char check according to the char length in the database and throw a exception!